ubyssey/templates/headers/topbar.html

```diff
@@ -21,7 +21,7 @@
                 </li><li {% if section == 'events' %}class="current"{% endif %}>
                     <a href="{% url 'events' %}">Events</a>
                 </li><li {% if section == 'podcast' %}class="current"{% endif %}>
-                    <a href="{% url 'podcasts' 'episodes' %}">Podcast</a>
+                    <a href="{% url 'podcasts' 'episodes' %}">Podcast <span class='upper-right-decorator'>New</span></a>
```

Comment:
Can we also add a podcast link in a mobile header as described in https://github.com/ubyssey/ubyssey.ca/issues/593 ?